README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Build Status 
 
-OV Version | 6.00 | 5.60 | 5.50 | 5.40 |
+OV Version | 6.10 | 6.00 | 5.60 | 5.50 |
 | ------------- |:-------------:| -------------:| -------------:| -------------:|
-SDK Version/Tag | [v6.0.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v6.0.0) | [v3.8.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v3.8.0) | [v3.7.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v3.7.0) | [v3.6.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v3.6.0) |
-Build Status | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)|
+SDK Version/Tag | [v6.1.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v6.0.0) | [v6.0.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v6.0.0) | [v3.8.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v3.8.0) | [v3.7.0](https://github.com/HewlettPackard/oneview-chef/releases/tag/v3.7.0) |
```

Comment:
for v6.1.0 tag, you have provided 6.0.0 url. please change

Reply:
Done